platform-tests/broker-acceptance/cdn_broker_test.go

```diff
@@ -88,16 +88,14 @@ var _ = Describe("CDN broker", func() {
 		})
 
 		It("refuses to create a CDN for a domain without a cf create-domain", func() {
+			orgName := testContext.TestSpace.OrganizationName()
 			domainName := generator.PrefixedRandomName(testConfig.GetNamePrefix(), "cdn-broker") + ".net"
 			domainNameList := fmt.Sprintf(`{"domain": "%s"}`, domainName)
 
 			serviceInstanceName = generator.PrefixedRandomName(testConfig.GetNamePrefix(), "test-cdn")
 
-			// best effort tidyup - we don't really care if these pass or fail.
-			// currently this kind of failure doesn't actually stop the service
-			// being "created".
-			defer pollForServiceDeletionCompletion(serviceInstanceName)
-			defer cf.Cf("delete-service", serviceInstanceName, "-f")
```

Comment:
By removing the delete-service are we risking orphaned cloudfront distributions if something isn't right? Should we not do a delete-service and if that fails then the instance purge?

Reply:
I think they never get deleted anyway that is why we are purging them. Malcolm would have more info on that.

Reply:
We can merge this for now. I will have a check on this change after it has been merged.